styles/main.css

```diff
@@ -258,18 +258,21 @@ img {
 }
 table{
     border-collapse: collapse;
-    background: rgba(0, 0, 0, 0.1);
-    padding-bottom: 3em;
-    padding-top: 3em;
+}
+td,
+th {
+    background: rgba(0, 0, 0, 0.05);
+    padding-bottom: 0.3em;
+    padding-top: 0.3em;
     padding-left: 1em;
     padding-right: 1em;
-    border-left: 3px solid #0bdd00;
-    border-right: 3px solid #0bdd00;
+    border-left: 3px solid #ffe4e1;
+    border-right: 3px solid #ffe4e1;
 }
 th {
-    background: rgba(0, 0, 0, 0.2);
+    background: rgba(0, 0, 0, 0.1);
 }
-tr:nth-child(odd) {background: rgba(0, 0, 0, 0.2);}
+tr:nth-child(odd) td {background: rgba(0, 0, 0, 0.1);}

```

Comment:
Note the added `td` here:  this styles the CELLS in odd rows rather than styling the odd ROWS as a whole